app/helpers/users_helper.rb

```diff
@@ -4,4 +4,8 @@ module UsersHelper
   def user_initials(user)
     user.full_name.remove(/(\(|\[).*(\)|\])/).scan(/([[:word:]])[[:word:]]+/i).join
   end
+
+  def user_can_add_new_member(user)
+    user.has_role?(:admin) || user.has_role?(:volunteer)
```

Comment:
This will raise ```undefined method `has_role?' for nil:NilClass``` if user is not logged in(`current_user = nil`).
At the moment we are rendering same nav bar for user and guest.

Reply:
is this fixed by adding the `if user_signed_in?` back to the header, or is this something I need to fix inside of the helper?

Reply:
I would rather go inside helper so we have view that has less branching and it's easier to read. But I'm merging this as is its ok....feel free to change it if you want in a new small PR :)